gatsby/content/projects/clients/watch-the-matrix.mdx

```diff
@@ -0,0 +1,57 @@
+---
+layout: projectimage
+title: Watch The Matrix
+categories:
+    - client
+description: A Matrix client for watchOS written in Swift
+author: pixlwave
+home: https://github.com/pixlwave/Watch-The-Matrix
+language: Swift
+license: MPL-2.0
+repo: https://github.com/pixlwave/Watch-The-Matrix
+room: "#watchthe:matrix.org"
+thumbnail: /docs/projects/images/wtm-small.png
+screenshot: /docs/projects/images/wtm-large.png
+maturity: Alpha
+platforms:
+    - watchOS
+featured: true
```

Comment:
AIUI `featured` just means it will be shown on the client matrix? Definitely not looking for anything more than that 🫣

Reply:
Off the top of my head it means it would be highlighted in https://matrix.org/docs/projects/try-matrix-now/, but our website is a bit of a mess atm and this is being reworked. If the maturity is Alpha, the client should probably not be featured :)

Reply:
Ohh, ha definitely not, no! Updated 👍

Reply:
FWIW I built this locally and it does not show up on `/clients-matrix` unless `featured: true`.

Reply:
Looks like this got merged, but I'd be happy to open another PR with `featured: true` and with a watchOS section to `/clients`. What say ye @thibaultamartin?

Reply:
Worry not, this is just current matrix.org being a bit of a crufty embarrassment. Watch the Matrix appears in https://matrix.org/docs/projects/try-matrix-now/

I'm mostly trying to keep the data sane, preparing for the new matrix.org. This client not being featured sounds like the sanest option, if it's not ready for prime time :)

Reply:
Ok, sounds good to me 👍